Library/Formula/dnscrypt-proxy.rb

```diff
@@ -41,6 +42,31 @@ def install
     system "make", "install"
   end
 
+  def post_install
+    if build.with? "minisign"
+      (bin/"dnscrypt-update-resolvers").write <<-EOS.undent
```

Comment:
You should write this script inside `def install` with the `chmod`, and then just execute it `postinstall` unless they've build without `minisig`. Otherwise you're going to run into things like this:

```
Error: Will not overwrite /usr/local/Cellar/dnscrypt-proxy/1.6.0_2/bin/dnscrypt-update-resolvers
```